elasticsearch/mapping_sirene_index.py

```diff
@@ -57,6 +57,8 @@
     ],
 )
 
+ELASTIC_SHARDS = 1
```

Comment:
maybe variabilize this but hey this is not the priority

Reply:
I'll do it in a separate PR. I did not want to overload this one even more than I already did 👉👈